src/components/Filters/Filters.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Gender from './Categories/Gender';
+import Status from './Categories/Status';
+import { Box, Button } from '@mui/material';
+
+const Filters = ({ gender, setGender, status, setStatus, setSearch }) => {
```

Comment:
I feel like `Filters` should be a pure presentation component, I'd change the props names to `gender, onGenderChange`
And take the `clearAll` logic outside this component, so you'd add a `onClearAllClick` prop